RevenueCatUI/CustomerCenter/Data/LoadPromotionalOfferUseCase.swift

```diff
@@ -45,14 +45,14 @@ class LoadPromotionalOfferUseCase: LoadPromotionalOfferUseCaseType {
 
             guard let productIdentifier = customerInfo.earliestExpiringAppStoreEntitlement()?.productIdentifier,
                   let subscribedProduct = await self.purchasesProvider.products([productIdentifier]).first else {
-                Logger.warning(Strings.could_not_offer_for_active_subscriptions)
+                Logger.warning(Strings.could_not_offer_for_any_active_subscriptions)
                 return .failure(CustomerCenterError.couldNotFindSubscriptionInformation)
             }
 
             guard let discount = subscribedProduct.discounts.first(where: {
                 $0.offerIdentifier == promoOfferDetails.iosOfferId
             }) else {
-                Logger.warning(Strings.could_not_offer_for_active_subscriptions)
+                Logger.warning(Strings.could_not_offer_for_active_subscriptions(promoOfferDetails.iosOfferId, productIdentifier))
```

Comment:
Hmm I'm not sure if these should be warnings or `debug`... seems like a case that may happen somewhat often if the customer is not eligible by apple? Not a strong opinion though

Reply:
> seems like a case that may happen somewhat often if the customer is not eligible by apple?

That will actually fail later, this will fail if the active product's discounts don't match with the promotional offers in the Customer Center config.

I actually think it could be a debug yes, I will change it